gapis/gfxapi/gles/gles.api

```diff
@@ -243,6 +243,146 @@ enum DeclaringApi {
+  bool GL_OES_point_size_array                         = true
+  bool GL_OES_primitive_bounding_box                   = true
+  bool GL_OES_query_matrix                             = true
+  bool GL_OES_sample_shading                           = true
+  bool GL_OES_single_precision                         = true
+  bool GL_OES_tessellation_shader                      = true
+  bool GL_OES_texture_3D                               = true
+  bool GL_OES_texture_border_clamp                     = true
+  bool GL_OES_texture_buffer                           = true
+  bool GL_OES_texture_cube_map                         = true
+  bool GL_OES_texture_half_float                       = true
+  bool GL_OES_texture_storage_multisample_2d_array     = true
+  bool GL_OES_texture_view                             = true
+  bool GL_OES_vertex_array_object                      = true
+  bool GL_OES_vertex_half_float                        = true
+  bool GL_OVR_multiview                                = true
+  bool GL_QCOM_alpha_test                              = true
+  bool GL_QCOM_driver_control                          = true
+  bool GL_QCOM_extended_get                            = true
+  bool GL_QCOM_extended_get2                           = true
+  bool GL_QCOM_tiled_rendering                         = true
+  bool GL_OES_compressed_ETC1_RGB8_texture             = true
+  bool GL_EXT_texture_filter_anisotropic               = true
+  bool GL_EXT_texture_norm16                           = true
+  bool GL_EXT_texture_sRGB_decode                      = true
+  bool GL_OES_depth_texture                            = true
+  bool GL_EXT_sRGB_write_control                       = true
+  bool GL_EXT_texture_rg                               = true
+  bool GL_EXT_texture_compression_s3tc                 = true
+  bool GL_OES_packed_depth_stencil                     = true
+  bool GL_EXT_texture_format_BGRA8888                  = true
+  bool GL_OES_viewport_array                           = true
+  bool GL_EXT_shader_pixel_local_storage2              = true
+  bool GL_EXT_clear_texture                            = true
+  bool GL_NV_conservative_raster_pre_snap_triangles    = true
+  bool GL_EXT_draw_transform_feedback                  = true
+  bool GL_IMG_framebuffer_downsample                   = true
+  bool GL_OVR_multiview_multisampled_render_to_texture = true
+  bool GL_IMG_bindless_texture                         = true
+  bool GL_NV_gpu_shader5                               = true
+  bool GL_EXT_polygon_offset_clamp                     = true
+  bool GL_NV_viewport_swizzle                          = true
+  bool GL_EXT_window_rectangles                        = true
+  bool GL_OES_EGL_image_external                       = true
+  bool EGL_KHR_image_base                              = true
+  bool EGL_ANDROID_create_native_client_buffer         = true
+  bool EGL_ANDROID_image_native_buffer                 = true
+}
+AvailableApis Api
```

Comment:
API is so overloaded, I'm slightly concerned with adding yet another definition to the codebase.
Part of this is GL version, part of it is extension. Can they not be split into two categories?